carbon-design-system/carbon-addons-iot-react
# fix(page-title-bar): incorrect truncation

Closes #2322 

**Summary**

- The Page title was truncating incorrectly when the `content` prop had _very_ wide content in it. This updates the CSS to prevent that from happening

**Change List (commits, features, bugs, etc)**

- updated css
- updated the story to show breaking scenario

**Acceptance Test (how to verify the PR)**

- go to [PageTitleBar `base` story](https://deploy-preview-2978--carbon-addons-iot-react.netlify.app/?path=/story/1-watson-iot-pagetitlebar--base)
- add extra characters to the Page title
- shrink window
- title should not truncate

**Regression Test (how to make sure this PR doesn't break old functionality)**

- confirm expected behavior is still the same on other PageTitleBar stories



**Things to look for during review**
